OpenRA.Game/Traits/Player/FrozenActorLayer.cs

```diff
@@ -25,6 +26,7 @@ public class FrozenActorLayerInfo : ITraitInfo
 	public class FrozenActor
 	{
 		public readonly MPos[] Footprint;
+		public readonly CPos[] OccCells;
```

Comment:
But that method cannot be used if the actor is null. A frozen actor can exist even after its actor is destroyed. The actor's occupied cells have to be stored in the frozen actor so that they can be used even if the actor is null.